cloud/deployment/deployment.go

```diff
@@ -63,12 +63,11 @@ const (
 )
 
 var (
-	sleepTime                  = 180
-	tickNum                    = 10
-	timeoutNum                 = 180
-	listLimit                  = 1000
-	dedicatedDeploymentRequest = astroplatformcore.UpdateDedicatedDeploymentRequest{}
```

Comment:
dropped the global variable, this makes no difference from CLI functionality pov, but it messes up the unit tests, since the global variables are shared